newapi/ooniapi/citizenlab.py

```diff
@@ -488,7 +497,17 @@ def url_submission_add_url():
             comment:
               type: string
             new_entry:
-              type: array
+              type: object
+              properties:
+                category_code:
+                  type: string
+                url:
+                  type: string
+                date_str:
```

Comment:
It's called `date_added` in the citizenlab test-lists repo.

Reply:
Fixed